pkg/api/client.go

```diff
@@ -18,3 +24,15 @@ func NewClient(endpoint string) *Client {
 		Endpoint:   endpoint,
 	}
 }
+
+func DefaultClientSetup(cmd *cobra.Command) (context.Context, *client.APIClient, error) {
+	endpointURL := cmd.Flags().Lookup("endpoint").Value.String()
+	endpointKey, err := EndpointURLToKey(endpointURL)
+	if err != nil {
+		return nil, nil, err
+	}
+	configuration := client.NewConfiguration()
+	apiClient := client.NewAPIClient(configuration)
+	context := context.WithValue(context.Background(), client.ContextServerIndex, endpointKey)
+	return context, apiClient, nil
```

Comment:
Why return the context? The background context should be handled by the API if for some reason it had a panic or exited early. And our control flow shouldn't be handling the background context anyways.

I wonder if it'd be better to do `context.TODO` since we're not really handling different context's control flow right now.